doc/api/buffer.md

```diff
@@ -3140,12 +3140,24 @@ added: v8.2.0
 #### `buffer.constants.MAX_LENGTH`
 <!-- YAML
 added: v8.2.0
+changes:
+  - version: v15.0.0
+    pr-url: https://github.com/nodejs/node/pull/35415
+    description: Value is changed to 2<sup>32</sup> on 64-bit
+      architectures.
```

Comment:
Usually we don't backport history entries that are "in the future", the first entry is supposed to reflect the current state for this release line.
```suggestion
```

Reply:
Thank you! Fixed, I totally forgot